networkit/cpp/distance/SSSP.cpp

```diff
@@ -17,7 +17,11 @@ SSSP::SSSP(const Graph &G, node source, bool storePaths,
       storeNodesSortedByDistance(storeNodesSortedByDistance) {}
 
 std::vector<edgeweight> SSSP::getDistances(bool moveOut) {
-    return (moveOut) ? std::move(distances) : distances;
```

Comment:
This never actually worked to begin with: the common type of `std::vector<edgeweight> &&` and `std::vector<edgeweight> &` is again `std::vector<edgeweight> &`...